Automattic/wp-calypso
# JPC: Remove plan tabs test

This PR remove the tabbed plans page in JPC introduced in https://github.com/Automattic/wp-calypso/pull/10695. The test went fine, but the results are not good enough to justify a change.

How to test
==========
1. Connect a jetpack site that doesn't have an associated plan. Once you get to the plans selection page, you should see this:
![image](https://cloud.githubusercontent.com/assets/1554855/23209850/4421d484-f8fb-11e6-8eca-1b73d3f610ee.png)
2. Select the free plan
3. Disconnect the site and connect it again.
4. Select any of the paid plans and make sure you can buy it.
